packages/blade/src/components/Accordion/AccordionButton.native.tsx

```diff
@@ -73,11 +73,16 @@ const _AccordionButton = ({ index, icon: Icon, children }: AccordionButtonProps)
     isPressed.value = pressed;
 
     const iconColor: IconProps['color'] = pressed
-      ? 'surface.action.icon.focus.lowContrast'
-      : 'surface.action.icon.default.lowContrast';
+      ? 'interactive.icon.gray.subtle'
+      : 'surface.text.gray.muted';
```

Comment:
this shall be icon right why is it text?

Reply:
Fixed